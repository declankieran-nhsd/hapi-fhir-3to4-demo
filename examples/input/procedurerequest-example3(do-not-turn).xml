<?xml version="1.0" encoding="UTF-8"?><ProcedureRequest xmlns="http://hl7.org/fhir">
	<!--   placer identifier  -->
	<text><status value="generated"/><div xmlns="http://www.w3.org/1999/xhtml"><p><b>Generated Narrative with Details</b></p><p><b>id</b>: do-not-turn</p><p><b>identifier</b>: 20170201-0002</p><p><b>status</b>: active</p><p><b>intent</b>: order</p><p><b>priority</b>: stat</p><p><b>doNotPerform</b>: true</p><p><b>code</b>: Turning patient in bed (procedure) <span>(Details : {SNOMED CT code &#39;359962006&#39; = &#39;Turning patient in bed&#39;, given as &#39;Turning patient in bed (procedure)&#39;})</span></p><p><b>subject</b>: <a>Patient/example</a></p><p><b>authoredOn</b>: 01/02/2017 5:23:07 PM</p><h3>Requesters</h3><table><tr><td>-</td><td><b>Agent</b></td></tr><tr><td>*</td><td><a>Dr Adam Careful</a></td></tr></table><p><b>reasonReference</b>: Patient has a spinal fracture</p></div></text><identifier>
		<system value="http://goodhealth.org/placer-ids">
		</system>
		<value value="20170201-0002"/>
	</identifier>
	<status value="active"/>
	<intent value="order"/>
	<priority value="stat"/>
	<doNotPerform value="true"/>
	<code>
		<coding>
			<system value="http://snomed.info/sct"/>
			<code value="359962006"/>
			<display value="Turning patient in bed (procedure)"/>
		</coding>
	</code>
	<subject>
		<reference value="Patient/example"/>
	</subject>
	<!--   TimeStamp when order authorized  -->
	<authoredOn value="2017-02-01T17:23:07Z"/>
	<requester>
		<agent>
			<reference value="Practitioner/example"/>
			<display value="Dr Adam Careful"/>
		</agent>
	</requester>
	<!--   demonstrating in this example the reason reference element   -->
	<reasonReference>
		<display value="Patient has a spinal fracture"/>
	</reasonReference>
</ProcedureRequest>