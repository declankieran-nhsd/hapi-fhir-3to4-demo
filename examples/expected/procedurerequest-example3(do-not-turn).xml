<?xml version="1.0" encoding="UTF-8"?>

<ServiceRequest xmlns="http://hl7.org/fhir">
  <text>
    <status value="generated"/><div xmlns="http://www.w3.org/1999/xhtml"><p><b>Generated Narrative with Details</b></p><p><b>id</b>: do-not-turn</p><p><b>identifier</b>: 20170201-0002</p><p><b>status</b>: active</p><p><b>intent</b>: order</p><p><b>priority</b>: stat</p><p><b>doNotPerform</b>: true</p><p><b>code</b>: Turning patient in bed (procedure) <span>(Details : {SNOMED CT code '359962006' = 'Turning patient in bed', given as 'Turning patient in bed (procedure)'})</span></p><p><b>subject</b>: <a>Patient/example</a></p><p><b>authoredOn</b>: 01/02/2017 5:23:07 PM</p><h3>Requesters</h3><table><tr><td>-</td><td><b>Agent</b></td></tr><tr><td>*</td><td><a>Dr Adam Careful</a></td></tr></table><p><b>reasonReference</b>: Patient has a spinal fracture</p></div>
  </text>
  <extension url="http://hl7.org/fhir/3.0/StructureDefinition/BaseType">
    <valueString value="ProcedureRequest"/>
  </extension>
  <identifier>
    <system value="http://goodhealth.org/placer-ids"/>
    <value value="20170201-0002"/>
  </identifier>
  <status value="active"/>
  <intent value="order"/>
  <priority value="stat"/>
  <doNotPerform value="true"/>
  <code>
    <coding>
      <system value="http://snomed.info/sct"/>
      <code value="359962006"/>
      <display value="Turning patient in bed (procedure)"/>
    </coding>
  </code>
  <subject>
    <reference value="Patient/example"/>
  </subject>
  <authoredOn value="2017-02-01T17:23:07Z"/>
  <requester>
    <reference value="Practitioner/example"/>
    <display value="Dr Adam Careful"/>
  </requester>
  <reasonReference>
    <display value="Patient has a spinal fracture"/>
  </reasonReference>
</ServiceRequest>